Components/LiveMap.jsx

```diff
@@ -42,57 +42,58 @@ export default function LiveMap() {
         locationRef.push({
           latitude,
           longitude,
+          timestamp,
         });
       },
       error => console.log(error.message),
       {
         enableHighAccuracy: true,
         distanceFilter: 10,
-        interval: 10000,
-        fastestInterval: 5000,
+        interval: 1000,
+        fastestInterval: 1000,
       },
-      );
-      
-       // Set up a listener for real-time changes to the 'locations' node
-       const onValueChange = database()
-       .ref('locations')
-       .on('value', snapshot => {
-          const newLocations = snapshot.val();
-          // console.log(newLocations);
+    );
+
+    // Set up a listener for real-time changes to the 'locations' node
```

Comment:
I don't think we need this code that supposed to be setting the location from the firebase database in this component any longer, since this component is only concerned with the streamer's position.

Reply:
Let's test this out first, but I agree it should be good to remove!